### PR TITLE
ADCM-2048_tests remove xfail from test_on_tasks_in_header_popup

### PR DESCRIPTION
**Pull request by annademina**

### CODE REVIEW

File: tests/ui_tests/test_jobs_page.py [hunk 1]
```diff
@@ -296,9 +301,14 @@ def test_job_has_correct_info_in_header_popup(
         assert (
             cluster_page.header.get_failed_job_amount_from_header() == job_info['failed_jobs']
         ), f"Failed job amount should be {job_info['failed_jobs']}"
-        assert job_info['background'] in cluster_page.header.get_jobs_circle_color(), "Bell circle should be colored"
```
Comment (Sealwing) [inline tests/ui_tests/test_jobs_page.py:299]: Why is it removed?

Comment (Sealwing) [reply tests/ui_tests/test_jobs_page.py:299]: Found

File: tests/ui_tests/app/page/job/page.py [hunk 2]
```diff
@@ -48,7 +48,9 @@ def __init__(self, driver, base_url, job_id: int):
 
     @allure.step("Check title on the page")
     def check_title(self, expected_title: str):
-        assert self.find_element(ObjectPageLocators.title).text == expected_title
+        self.wait_element_visible(ObjectPageLocators.title)
+        current_title = self.find_element(ObjectPageLocators.title).text
+        assert current_title == expected_title, f"Should be title '{expected_title}', but was {current_title}''"
```
Comment (Sealwing) [inline tests/ui_tests/app/page/job/page.py:53]: For further occasions: format of "noun verb" is easier to read, don't you think?
No need to actually change it, but "Title should be 'smt', not 'smt else'" looks better

File: tests/ui_tests/test_jobs_page.py [hunk 3]
```diff
@@ -296,9 +301,14 @@ def test_job_has_correct_info_in_header_popup(
...
 
-    @pytest.mark.xfail(reason="https://arenadata.atlassian.net/browse/ADCM-2048")
+        def wait_for_background():
+            assert (
+                job_info['background'] in cluster_page.header.get_jobs_circle_color()
+            ), "Bell circle should be colored"
+
+        wait_until_step_succeeds(wait_for_background, period=1, timeout=10)
```
Comment (Sealwing) [inline tests/ui_tests/test_jobs_page.py:310]: Maybe check more frequently? Like 0.3-0.5 secs? Or from experience, it's not likely to be less than a second?

**Review by Sealwing — DISMISSED** [review]

File: tests/ui_tests/app/page/job/page.py (hunk 2 above)
Comment (annademina) [reply tests/ui_tests/app/page/job/page.py]: changed

File: tests/ui_tests/test_jobs_page.py (hunk 3 above)
Comment (annademina) [reply tests/ui_tests/test_jobs_page.py]: okay, changed)

**Review by Sealwing — APPROVED** [review]

**Review by skhomuti — APPROVED** [review]

**Pull request merged**